build/org.eclipse.cdt.managedbuilder.core.tests/tests/org/eclipse/cdt/managedbuilder/core/tests/BuildDescriptionModelTests.java

```diff
@@ -1755,11 +1749,6 @@ public void testDesAddlInVarUserObjs() {
 		//			type.addResource(tDes.createResource(cName + "/t.o"));
 		//			type.addResource(tDes.createResource(cName + "/y.o"));
 
-		//		type = step.createIOType(true, false, null);
-		type.addResource(tDes.createResource("o1.o"));
```

Comment:
Is there no need to check that these files gets handled somewhere else or is it enough to just drop the check?

Reply:
There is nothing further to check in the context of this test since the user objects are no-longer part of the ${INPUTS} of the linker.